docs/FUZZING.md

```diff
@@ -0,0 +1,79 @@
+# Fuzzing
+
+Fuzz testing is an automated software testing method that injects invalid, 
+malformed, or unexpected inputs to reveal defects and vulnerabilities. A fuzzing 
+tool monitors the system for exceptions like crashes, information leakage, or 
+errors, helping developers identify and fix bugs and security loopholes.
+
+## Current state of fuzzing in liboqs
+- [ ] kem 
+  - [ ] bike
+  - [ ] classic_mceliece
+  - [ ] frodokem
+  - [ ] hqc
+  - [ ] kyber
+  - [ ] ml_kem
+  - [ ] ntruprime
+- [ ] sig
+  - [ ] dilithium
+    - [ ] dilithium1
+    - [x] dilithium2
```

Comment:
```suggestion
    - [x] dilithium2
    - [ ] dilithium3
    - [ ] dilithium5
```

Reply:
Is there a dilithium4?

Reply:
I've incorporated this change into the original commit leaving out dilithium4, as after some searching it doesn't look like it exists.